usr/src/cmd/zpool/Makefile

```diff
@@ -29,6 +30,7 @@ POFILES=$(OBJS:%.o=%.po)
 POFILE= zpool.po
 
 include ../Makefile.cmd
+#include ../Makefile.cmd.64
```

Comment:
I have added commented line for information - how to build 64bit component.
I¹m using now 64bit and, probably some others can be interested in it too -
it will be easy uncomment it on personal fork and use.

If it is not good idea/way - I can remove it.

--
Best regards,
Igor Kozhukhov

From:  Matthew Ahrens <notifications@github.com>
Reply-To:  openzfs/openzfs
<reply+000e4693d72c6422ae882a785e51e93060b55a75dd81f8bb92cf0000000112ae3f549
2a170ce02f50e54@reply.github.com>
Date:  Wednesday 13 January 2016 at 19:37
To:  openzfs/openzfs <openzfs@noreply.github.com>
Cc:  Igor Kozhukhov <ikozhukhov@gmail.com>
Subject:  Re: [openzfs] 6551 cmd/zpool: cleanup gcc warnings (#57)

> 
> In usr/src/cmd/zpool/Makefile
> <https://github.com/openzfs/openzfs/pull/57#discussion_r49614420> :
>> > @@ -29,6 +30,7 @@ POFILES=$(OBJS:%.o=%.po)
>> >  POFILE= zpool.po
>> >  
>> >  include ../Makefile.cmd
>> > +#include ../Makefile.cmd.64
> what this the point of adding the commented out code?
> 
> 
> Reply to this email directly or view it on GitHub
> <https://github.com/openzfs/openzfs/pull/57/files#r49614420> .
> 
>